### ISSUE TITLE
aria-activedescendant used incorrectly in date picker

### ISSUE THREAD

**Issue by Rykus0** [amsul/pickadate.js#1012]
The value of aria-activedescendant should be the ID of the currently active element, which is a child of the element containing the aria-activedescendant element.  

https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/ARIA_Techniques/Using_the_aria-activedescendant_attribute
